style.css

```diff
@@ -759,6 +855,35 @@ header {
     text-align: center;
     padding: 12px 16px;
   }
+
+  /* Modal */
+  .popup-description {
+    display: flex;
+    justify-content: center;
+  }
+
+  .popup-description .card__skills {
```

Comment:
Please ensure your desktop modal design buttons matches that of the original Figma design 🙏.
Your design
![Screenshot_20230419_214132](https://user-images.githubusercontent.com/44624138/233196756-03436344-4518-47a1-91b5-71fc999d885d.jpg)
Figma design
![Screenshot_20230419_214411](https://user-images.githubusercontent.com/44624138/233196825-c0bc7ae8-61c4-4768-931c-544583ffa31b.jpg)